javascript/javascript2/week2/home-work.js

```diff
@@ -0,0 +1,63 @@
+//doubling the odd numbers
+const myarray = new Array(22, 79, 34, 17, 1095, 678, 345, 982, 899, 6784, 72837, 56783, 98986);
```

Comment:
this code is perfectly fine, however, you also can use modern code like
myArray = [12,12,34,56]

Reply:
👍🏻